src/Shared/Resources/Strings.shared.resx

```diff
@@ -66,6 +66,9 @@
   <data name="ExpectedEventToBeSerializable" Visibility="Public">
     <value>Event type "{0}" was expected to be serializable using the .NET serializer. The event was not serializable and has been ignored.</value>
   </data>
+  <data name="DeprecatedEventSerialization" Visibility="Public">
+    <value>Usage of unsecure BinaryFormatter during serialization of custom event type '{0}'. This will be deprecated soon. Please use Extended*EventArgs instead. More info: https://aka.ms/msbuild/eventargs</value>
```

Comment:
The learn article probably needs update to mention the deprecation of `CustomBuildEventArgs` and their intended replacement.

Give @ghogen requirements on what/how needs to be publicly documented (or possibly create doc bug with details here https://github.com/dotnet/docs/issues) - so that this gets on documentation plan early enough

Reply:
Spined off as separate item: https://github.com/dotnet/msbuild/issues/9016